src/services/players/player.hpp

```diff
@@ -100,6 +101,11 @@ namespace big
 
 		int spectating_player = -1;
 
+		time_t ptfx_sent		= time(0);
+		int ptfx_lastsend		= 0;
+		bool cageNotificationSent = false;
+		bool ptfxSpamNotificationSent = false;
```

Comment:
These variables do not follow the naming scheme used in YimMenu, change them to `snake_case`.

Also see my comment on the `rate_limiter` making these variables obsolete.